.github/workflows/main.yml

```diff
@@ -152,29 +152,48 @@ jobs:
             dist
             packages
 
-  scanning:
-    name: Test GitHub secret action
+  test_github_secret_scan_action:
+    name: Test GitHub actions/secret
```

Comment:
```suggestion
    name: Test GitHub actions-unstable/secret
```

Reply:
Fixed

Reply:
I am not sure about this: the goal is really to test the actions in "actions/*", but due to the way our actions are defined, we can't test them directly, we have to use the "actions-unstable/*" actions instead. One of the reasons for generating the endpoint.sh files is to ensure "actions/$X" and "actions-unstable/$X" remains in sync (maybe we should also generate the Dockerfile and action.yml? I refrained from doing so, but did silly mistakes because of that 🤔)

Reply:
I'd say we start with this version that is already great. We'll see later if it is painful to maintain actions in sync ?    
About the wording, we could just say "Test GitHub action for secret" and "Test GitHub action for iac" ?